### ISSUE TITLE
Could not find hostfxr.dll

### ISSUE THREAD

**Issue by thewayur** [Tyrrrz/YoutubeDownloader#267]
### Version

1.9.2

### Details

it was running fine, suddenly got this issue.
so i downloaded fresh, and at time of launching, same error popped

![image](https://user-images.githubusercontent.com/2944767/171337387-99a25ff6-73ad-4707-95f8-6b108504f8ac.png)

### Steps to reproduce

at launch

**Comment by Tyrrrz** [comment]
Can you try uninstalling .NET Runtime if you have it and try running the app again?

**Comment by thewayur** [comment]
i had to install dotnet6 sdk manually from: [https://dotnet.microsoft.com/en-us/download/dotnet/6.0](url) 

direct link: [https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.300-windows-x64-installer](url)

because after upgrade/update of youtubedownloader it was relying on dotnet 4/5 or watever (i noticed it while previous update)

but when i freshly installed youtube downloader in freshly installed windows it started download dot net 6 automatically (which was failed by updating in previous computer)


so in short, if u r going to update current youtube downloader, it will not autodownload dotnet 6

thanks for ur programm, my problem is resolved!
